single-product.css

```diff
@@ -0,0 +1,78 @@
+.single-product-page-container .nav-bar {
+    margin-bottom: 2rem;
+}
+
+.single-product-page-container .sidebar-products-container {
+    display: flex;
+}
+
+.single-product-page-container .sidebar-products-container .sidebar-container {
+    min-width: 25rem;
+    padding: 0 3rem;
+    border-right:1px solid var(--button-floatHover-color);
+}
```

Comment:
You named class quite descriptive. For a reviewer, it is very convenient. Great work Jyotsna.